sympy/sets/tests/test_fancysets.py

```diff
@@ -163,7 +166,6 @@ def test_Range_set():
     raises(ValueError, lambda: Range(-oo, oo))
     raises(ValueError, lambda: Range(oo, -oo, -1))
     raises(ValueError, lambda: Range(-oo, oo, 2))
-    raises(ValueError, lambda: Range(0, pi, 1))
```

Comment:
maybe this should still raise an exception.

Reply:
It's well defined and not susceptible to confusion (I think).